pallets/manta-pay/Cargo.toml

```diff
@@ -87,35 +87,35 @@ precompute-coins = [
 # utils
 anyhow = { version = "1.0.55", optional = true, default-features = false }
 base64 = { version = "0.20", default-features = false, features = ["alloc"] }
-indoc = { version = "1.0.3", optional = true, default-features = false }
-rand_chacha = { version = "0.3.1", optional = true, default-features = false }
-tempfile = { version = "3.3.0", optional = true, default-features = false }
 
 # substrate dependencies
 frame-benchmarking = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.26", default-features = false, optional = true }
 frame-support = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.26", default-features = false }
 frame-system = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.26", default-features = false }
-scale-codec = { package = "parity-scale-codec", version = '3.1.2', default-features = false, features = ["derive", "max-encoded-len"] }
```

Comment:
keep order of dependencies

these seem to be correctly under the substrate dependencies